iron-selectable.html

```diff
@@ -159,6 +168,9 @@
         this._updateSelected();
       }
       this._addListener(this.activateEvent);
+      // Resolve timing issue where attached would not be called,
+      // but selected was update with to defaultSelected
```

Comment:
Can you clarify? What's exactly the timing issue here?

We also want to be sure that when `defaultSelected` is changed externally we also change `_defaultSelected`.